completions/ip

```diff
@@ -384,12 +384,30 @@ _comp_cmd_ip()
                 add | identify | list-id)
                     # TODO
                     ;;
-                delete | exec | pids | set)
+                delete | pids | set)
                     [[ $prev == "$subcmd" ]] && _comp_cmd_ip__netns "$1"
                     ;;
+                exec)
+                    local is_all=0
+                    for ((i = 1; i <= cword; i++)); do
+                        case ${words[i]} in
+                            -a | -all)
+                                is_all=1
+                                break
+                                ;;
+                        esac
+                    done
+                    if [[ $prev == "$subcmd" && $is_all != 1 ]]; then
+                        _comp_cmd_ip__netns "$1"
+                    else
+                        local offset i
```

Comment:
`i` should likely be localized earlier, see above.
```suggestion
                        local offset
```

Reply:
Yeah that's a mistake, fixed